### PR TITLE
Install yq for Dockerfile builds

### PR DESCRIPTION
**Pull request by coverbeck**
#3297 

Also added Quay.io badge to README.md

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
 FROM maven:3.6.2-jdk-11 AS maven
+RUN wget https://github.com/mikefarah/yq/releases/download/3.1.2/yq_linux_amd64 \
```
Comment (denis-yuen) [inline Dockerfile]: Why not use snap or apt?
https://github.com/mikefarah/yq#ubuntu-and-other-linux-distros-supporting-snap-packages

Comment (garyluu) [reply Dockerfile]: When I was doing this for CircleCI, snap didn't work for that image. Installing snap was not meant to be.  Tried to do this with apt also, ran into some other problems adding the repository.  Gave up and did it this way.

Comment (denis-yuen) [reply Dockerfile]: Fair enough

Comment (coverbeck) [reply Dockerfile]: I copied this from config.yml (which is another issue, that we have it in two places, but I don't know how to solve that off-hand).

**Pull request merged**